hack/patches/0001-Patch-to-fix-generate-groups.sh.patch

```diff
@@ -0,0 +1,40 @@
+From 682f06406c73726775c973f9abc72a0b80a97bad Mon Sep 17 00:00:00 2001
```

Comment:
ideally we remove the patch, asap.

should we track it somewhere, to remove ? 

e.g. GH issue ?